### PR TITLE
Google Summer of Code’22: Port fortran-lang.org to Sphinx

### LINKED ISSUE #56: typos
Replace "differnet" with "different" at https://fortran-lang.org/learn/best_practices/arrays . The sentence should be changed from

"This easily allows to create more generic functions that have to deal with differnet array ranks."

to

"This allows you to easily create more generic functions that can deal with different array ranks."

### LINKED ISSUE #115: Automatically delete previews on merge
Can we adapt the preview workflow to automatically remove previews once a PR is merged? Otherwise we will end up with several stale previews which have to be deleted from time to time.

### LINKED ISSUE #116: Reduce diff between sphinx-intl and weblate modified po-files
Updating the po-files with the `sphinx-intl` results in a huge diff due to changes in the maximum line length. We should investigate whether there is an option to get this consistent between weblate and sphinx-intl to avoid generating useless diffs.

### LINKED ISSUE #118: Broken format in intrinsics
Looks like we need to escape the exponential operator `**` as `\*\*` or use backticks ``` ` ``` here.

image

### PR DESCRIPTION
**Pull request by henilp105**
This PR is to port the Existing site of fortran-lang.org to sphinx.
This GSoC’22 project is done by Henil Shalin Panchal under the Guidance of GSoC mentors Sebastian Ehlert @awvwgk and Rohit Goswami @HaoZeke .I request all the members of the organisation fortran-lang https://github.com/orgs/fortran-lang/teams/admins https://github.com/fortran-lang please review this PR and add suggestions and comments on this PR.

https://summerofcode.withgoogle.com/programs/2022/projects/L28fukIz

This PR Fixes the Following Issues :
- [x] resolves #56
- [x] resolves #115
- [x] resolves #116
- [x] resolves rss feed url on nav bar.
- [x] prettified files.
- [x] resolves #118

Thanks and Regards,
Henil Shalin Panchal

CC @awvwgk

### CODE REVIEW

File: .github/workflows/fortran_packages.yml [hunk 1]
```diff
@@ -28,5 +28,5 @@ jobs:
       - name: Upload data files
         uses: actions/upload-artifact@v3
         with:
-          name: _data
-          path: _data/
+          name: data
+          path: data/
```
Comment (awvwgk) [inline .github/workflows/fortran_packages.yml:32]: Can we keep this at `_data/`? The `data/` directory should contain the human generated files, which are updated with the repository and the `_data/` the machine generated once. The cache might be out of sync between `data/` and `_data/` directory, but changes to languages would become available immediately without refreshing the cache.

Comment (henilp105) [reply .github/workflows/fortran_packages.yml:32]: Yes sir, sure will revert this changes.

**Comment by henilp105** [comment]
#build_preview

**Comment by henilp105** [comment]
#build_preview

**Review by awvwgk — APPROVED** [review]

**Comment by henilp105** [comment]
#delete_preview

**Comment by henilp105** [comment]
#delete_preview

**Pull request merged**